### ISSUE TITLE
contact form doesn't work

### ISSUE THREAD

**Issue by walkero-gr** [getgrav/grav-skeleton-agency-site#11]
Unfortunately, contact form doesn't work at all. I use the latest version of the skeleton.
When I test the email plugin from terminal, with the following command, the email is received just fine.
bin/plugin email .....

I do not have any error at the logs. Also, when I submit the contact form, it doesn't lead me to a thank you page.

Any clue what should be done?

**Comment by johannesE** [comment]
Check out this: https://github.com/getgrav/grav-theme-agency/pull/17
And this: https://github.com/getgrav/grav-theme-agency/issues/16
Sadly I still cannot get it to work.

**Comment by consumr** [comment]
I got this to work by adding:

        email:
            subject: "[Site Contact Form] {{ form.value.name|e }}"
            body: "{% include 'forms/data.html.twig' %}"

to the process section of the form.md, as is indicated in the [doc](https://learn.getgrav.org/16/forms/forms/example-form).

**Comment by erdnuesse** [comment]
nope, spent 2 days now trying to fix the contact form, no success. 
pretty much abandonware since 2017.
